ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -102,6 +113,78 @@ def cleanup(cluster_name, cluster_id, upi=False):
         destroy_cluster(installer=oc_bin, cluster_path=cleanup_path)
 
 
+def cycle_nodes(cluster_path, action):
```

Comment:
I don't think the `cleanup` module is the right place for this code as they aren't really dealing with the cleanup of any resources. I think the [cluster module](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py) might make more sense?

Reply:
I can move it to utils as its not really a cluster method just a helper function.

Reply:
I think cluster pause is big enough to have its own module. This is adding an entirely new CLI entry point and shouldn't be "hidden away" in the utils module.

Reply:
Its in separate module now.